test/e2e/lib/gutenberg/blocks/gallery-masonry-block-component.js

```diff
@@ -12,6 +12,23 @@ class GalleryMasonryBlockComponent extends GutenbergBlockComponent {
 	static blockTitle = 'Masonry';
 	static blockName = 'coblocks/gallery-masonry';
 	static blockFrontendSelector = By.css( '.entry-content .wp-block-coblocks-gallery-masonry' );
+
+	/**
+	 * Uploads images to the gallery.
+	 *
+	 * NOTE/TODO This is common to a few gallery-like blocks. DRY it using a mixin or common base class.
+	 *
+	 * @param {{imageName: string, fileName: string, file: string}} filesDetails a list of fileDetails
+	 */
+	async uploadImages( filesDetails ) {
+		const fileInputSelector = By.css( `div[id="${ this.blockID.slice( 1 ) }"] input[type=file]` );
```

Comment:
`div[id='${ this.blockID.slice( 1 ) }']` can be just `this.blockID` (see [this comment](https://github.com/Automattic/wp-calypso/pull/45796#discussion_r500856393)).